jina/executors/clients.py

```diff
@@ -125,7 +126,7 @@ def get_response(self, request: 'predict_pb2.PredictRequest'):
             raise ValueError
         return self.get_output(_response)
 
-    def get_output(self, response: grpc.UnaryUnaryMultiCallable):
+    def get_output(self, response: grpc.UnaryUnaryMultiCallable) -> None:
```

Comment:
not really convinced if this is true for child classes, and I think in these cases, interface type is cleaner if provided by the parent